### PR TITLE
feat: theme settings and textStyle plugin

### PR DESCRIPTION
**Pull request by ragi96**
get rid of design tokens, and inject the TextStyles into the text style plugin (need to release fondue first)

Changes are related to https://github.com/Frontify/fondue/pull/1419

### CODE REVIEW

File: packages/callout-block/src/CalloutBlock.tsx [hunk 1]
```diff
@@ -5,41 +5,40 @@ import '@frontify/fondue-tokens/styles';
...
+                return style.getPropertyValue('--f-theme-settings-accent-color-info-color');
             case Type.Note:
-                return designTokens?.callout?.note;
+                return style.getPropertyValue('--f-theme-settings-accent-color-note-color');
             case Type.Tip:
-                return designTokens?.callout?.tip;
+                return style.getPropertyValue('--f-theme-settings-accent-color-tip-color');
             case Type.Warning:
-                return designTokens?.callout?.warning;
+                return style.getPropertyValue('--f-theme-settings-accent-color-warning-color');
```
Comment (fulopdaniel) [inline packages/callout-block/src/CalloutBlock.tsx:41]: I wonder if `f-theme-settings` prefix should be exported from Fondue.

**Review by fulopdaniel — APPROVED** [review]

**Comment by fulopdaniel** [comment]
Very impressive work 👏  The PR looks good to me.

File: packages/animation-curve-block/src/AnimationCurveBlock.spec.ct.tsx [hunk 2]
```diff
@@ -35,8 +35,8 @@ class AnimationCurveDummy {
     static with(id?: string): AnimationCurve {
         return {
             id: id ?? '1',
-            title: convertToRteValue(TextStyles.ELEMENT_HEADING3, 'Animation Title'),
-            description: convertToRteValue(TextStyles.ELEMENT_PARAGRAPH, 'Animation Description'),
+            title: convertToRteValue(TextStyles.heading3, 'Animation Title'),
+            description: convertToRteValue('p', 'Animation Description'),
```
[the remark this answers is not in the document]
Comment (ragi96) [reply packages/animation-curve-block/src/AnimationCurveBlock.spec.ct.tsx:39]: no, in the long run we won't even have that `TextStyles` anymore (probably we will add our own within shared)

**Review by celine-s — COMMENTED** [review]
noice!

Found just the issue with the main_font we shortly discussed this morning.

File: packages/shared/src/helpers/mapToGuidelineDesignTokens.ts [hunk 3]
```diff
@@ -1,143 +0,0 @@
...
-    key: string,
-    cssStyles: TokenValues,
-    value: string,
-    mainFontFamily: string,
-    fontFamilyCss?: string
-) => {
-    cssStyles.hover = cssStyles.hover || {};
-    switch (key) {
-        case DesignTokenPropertiesEnum.family:
-            cssStyles.fontFamily = getFont(fontFamilyCss ?? value, mainFontFamily);
```
[the remark this answers is not in the document]
Comment (ragi96) [reply packages/shared/src/helpers/mapToGuidelineDesignTokens.ts:57]: Thanks for finding that bug, needs to be fixed within web-app: https://github.com/Frontify/web-app/pull/116

[the remark this answers is not in the document]
Comment (celine-s) [reply packages/shared/src/helpers/mapToGuidelineDesignTokens.ts:57]: perfect 👍

**Review by celine-s — APPROVED** [review]
nice!

**Pull request merged**